src/components/Arrow/Arrow.module.scss

```diff
@@ -20,3 +20,7 @@
 .arrow {
   @include arrow($blue);
 }
+
+.branchArrow {
+  @include arrow($red);
+}
```

Comment:
The naming here is a little confusing. Its not that all branch arrows are red but rather selected/current/highlighted branch arrows are red. 
EDIT: it looks like this naming convention carries down into JS variables as well which should be updated